x-pack/plugins/rollup/public/crud_app/constants/validation.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { INDEX_ILLEGAL_CHARACTERS_VISIBLE } from 'ui/indices';
```

Comment:
isn't this `INDEX_ILLEGAL_CHARACTERS_VISIBLE` export only used for validating index names? (I believe illegal characters for index _patterns_ are in a different export)

if so, can we just add `*` to this export in `ui/indices` so that it takes effect everywhere we validate index names?

Reply:
ah ok. thanks for pointing this out. yeah, there's also `'ui/index_patterns'`, which is used for validating the `index pattern` field. 

i noticed `ui/indices` is being used in a couple different places so wasn't sure if it was a safe assumption to update for everything. i'll do some more investigation.

Reply:
Good call! You're right.

Reply:
thanks, @cjcenizal and @jen-huang! i updated the code if you could take another look when you have a chance.